youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,63 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?(\?.*)?$'
```

Comment:
Okay, I see what you mean. I'd argue that's different as I could (should?) have just as well left it out.